pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2ClientConfiguration.java

```diff
@@ -431,10 +417,9 @@ public boolean isAuthnRequestSigned() {
 	 *            Web context to transport additional information to the configuration.
 	 */
     protected void init(final String clientName, final WebContext context) {
```

Comment:
Thank you for letting me know in advance! I think I will get to the update of the extension after PAC 2.0 is released, not before. Thanks!